cli.js

```diff
@@ -22,6 +22,7 @@ const cli = meow(`
 	Options
 	  --copy -c       Copy the first emoji to the clipboard
 	  --skin-tone -s  Set and persist the default emoji skin tone (0 to 5)
+	  --limit -l      Maximum number of emojis to display (default: 7)
```

Comment:
It's not obvious from the description that the value is persisted. Does it even make sense to persist it?

Reply:
Ya, I guess we can ignore the persistence of the value.